src/server/routes/contract/extensions/erc721/read/signaturePrepare.ts

```diff
@@ -300,73 +335,9 @@ export async function erc721SignaturePrepare(fastify: FastifyInstance) {
   });
 }
 
-type GenerateMintSignaturePayloadOptions = Omit<
```

Comment:
Removed this unnecessary abstraction. The main handler function now does all transformations on the input fields.